services/collections-publisher/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.3
+
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y vim
+RUN apt-get install -y libxss1 libappindicator1 libindicator7
+RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && apt install -y ./google-chrome*.deb
+
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update -qq && apt-get install -y yarn nodejs
+
+RUN apt-get clean
```

Comment:
Apparently we don't need this because official docker images like `ruby` [automatically run it (at the bottom of the section in a call out)](https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#apt-get)

Reply:
👍